src/core_seaice/shared/mpas_seaice_berg_velocity_solver.F

```diff
@@ -814,6 +813,40 @@ subroutine berg_vertical_areas(domain)
 
        enddo ! iCell
 
+       block => block % next
```

Comment:
A separate block loop was needed in order to do a halo exchange before using these quantities.